CONTRIBUTING.md

```diff
@@ -178,6 +184,7 @@ Mainly for markup languages.
 ```
 @scope
 @reference
+@constructor
```

Comment:
Looks like we have `@constructor` already in line ~119

Reply:
Yes, but you can use it in both highlights and locals, thus the duplication.